test/zipkin/util.go

```diff
@@ -171,7 +175,7 @@ func jsonTrace(traceID string) ([]model.SpanModel, error) {
 	var models []model.SpanModel
 	err = json.Unmarshal(body, &models)
 	if err != nil {
-		return empty, err
+		return empty, fmt.Errorf("unmarshalling JSON, %q: %v", body, err)
```

Comment:
```suggestion
		return empty, fmt.Errorf("got an error in unmarshalling JSON %q: %v", body, err)
```

Reply:
Done, PTAL.

Although I go back and forth on this. In the future, I think I will try to follow https://github.com/uber-go/guide/blob/master/style.md#error-wrapping, where essentially I don't add the 'failed to' at every step and just let the final log line add the 'failed to' prefix.

Reply:
Thanks. That doc looks pretty useful BTW.